lib/scrabble.rb

```diff
@@ -10,4 +10,59 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
+
+
+    if word == nil
+      return nil
+    else
+      letters = word.chars
+
+      return_score = letters.map do |score|
+        score_it = score.upcase
+        point_values[score_it]
+      end
+    end
+
+    final_score = return_score.sum
+
+    final_score
+  end
+
+  def score_with_multipliers(word, letter_multiplier, word_multiplier = 1)
```

Comment:
The logic in this method looks great, but it seems like the method is doing _alot_ - try splitting these larger methods into smaller helper methods.